nautobot/dcim/tests/test_filters.py

```diff
@@ -1119,10 +1119,9 @@ def test_base_site(self):
 
     def test_subtree(self):
         params = {"subtree": [self.loc1.slug, self.nested_loc.pk]}
-        self.assertQuerysetEqualAndNotEmpty(
-            self.filterset(params, self.queryset).qs,
-            Location.objects.get(name=self.loc1.name).descendants(include_self=True),
-        )
+        expected = Location.objects.get(name=self.loc1.name).descendants(include_self=True)
+        expected |= Location.objects.get(name=self.nested_loc.name).descendants(include_self=True)
```

Comment:
Interesting. Not sure how this was ever passing as previously implemented. :-)

Reply:
Luck 😆